packages/grid/src/vaadin-grid-column.d.ts

```diff
@@ -28,14 +28,14 @@ export declare class ColumnBaseMixinClass<TItem> {
 
   /**
    * When true, the column is frozen. When a column inside of a column group is frozen,
-   * all of the sibling columns inside the group will get frozen also.
+   * all the sibling columns inside the group will get frozen also.
```

Comment:
If you're planning to change the documentation here, the same changes need to be applied in the js files also. Same applies to the other documentation changes below.

Reply:
Thanks, fixed in both places and made it nicer a bit more.